otx/algorithms/classification/utils/data.py

```diff
@@ -70,8 +73,10 @@ def __init__(
         val_data_root=None,
         test_ann_file=None,
         test_data_root=None,
+        ul_data_root=None,
```

Comment:
it's same as above comment. how about change this name to unlabeled_data_root?

Reply:
Fixed. Please check.